masschallenge/django-accelerator
# [AC-9358]: Company Profile Progress indicator

[AC-9358](https://masschallenge.atlassian.net/browse/AC-9358)

How to test:
- On your local machine, run `django-shell`
- Fetch any startup instance and call `startup.progress_complete()`
- You can save more data for for that instance based on the specified fields on [AC-9358](https://masschallenge.atlassian.net/browse/AC-9358) for either startup or business proposition to see how the progress increases and milestones change

NB: A quicker approach to testing is the follow my test cases and run in django-shell using test factories for all possible scenarios
